.github/pull_request_template.md

```diff
@@ -0,0 +1 @@
+DESCRIPTION: 
```

Comment:
We might also add a warning about the length of the description, which is 78 character.